.circleci/config.yml

```diff
@@ -301,18 +351,29 @@ workflows:
 
   test-build-deploy:
     jobs:
-      - test:
+      - setup_test_environment:
           <<: *all_tags
       - api_docs:
           <<: *all_tags
+          requires:
+            - setup_test_environment
+      - rspec_tests:
+          <<: *all_tags
+          requires:
+            - setup_test_environment
+      - rubocop_tests:
```

Comment:
```suggestion
      - linters:
```

Reply:
I **love** the rubocop separation. It'll give us much better feedback about what category of failures are happening.